SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Station.swift
+//  
+//
+//  Created by Carl Hill-Popper on 11/13/15.
+//  Copyright © 2015 Vokal.
+//
+
+import Foundation
+import CoreData
+import CoreLocation
+import Vokoder
+
+//lets pretend we're using mogenerator to create attribute constants
+enum StationAttributes: String {
+    case name
+    case longitude
+    case locationString
+    case latitude
+    case identifier
+    case descriptiveName
+    case accessible
+}
+
+@objc(Station)
```

Comment:
One thing to note about this for the README: If you use `Category`, it doesn't work at all in Objc becuase it thinks it's redefining something. :sad-trombone: